modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/server/gwt/BundleGWTServiceImpl.java

```diff
@@ -71,7 +71,7 @@ public ResourceTypeBundleConfiguration getResourceTypeBundleConfiguration(int co
     public BundleVersion createOrStoreBundleVersionViaURL(String url, String username, String password)
         throws RuntimeException {
         try {
-            BundleVersion results = bundleManager.createBundleVersionViaURL(getSessionSubject(), url, username,
+            BundleVersion results = bundleManager.createOrStoreBundleVersionViaURL(getSessionSubject(), url, username,
```

Comment:
I would NOT change API names like this. IIRC, that is a public API and if so I'm fairly certain customers will have used it. If anything, create a new method with the name you want, but maintain the old method for backward compatibility.